src/ray/stats/metric_defs.h

```diff
@@ -36,11 +36,11 @@ static Histogram GcsLatency("gcs_latency",
 /// Raylet Metrics
 ///
 static Gauge LocalAvailableResource("local_available_resource",
-                                    "The available resources on this node.", "pcs",
+                                    "The available resources on this node.", "",
```

Comment:
Is it possible to just remove the unit field entirely?

Reply:
I will actually createt a issue for this since I'd like to merge this ASAP before branch cut.